### PR TITLE
Add blog post for Velero 1.6

### PR DESCRIPTION
**Pull request by zubron**
Signed-off-by: Bridget McErlean <bmcerlean@vmware.com>

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: site/content/posts/2021-04-13-Velero-1.6-Bring-All-Your-Credentials.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+If you are interested in contributing more, or finding out what is involved in becoming a maintainer, please see our documentation on [project governance](https://github.com/vmware-tanzu/velero/blob/main/GOVERNANCE.md).
+
+### Improving Developer Experience
+
+As we continue to grow our community of contributors, we want to lower the barrier to entry for making contributions to the Velero project.
+We’ve made huge improvements to the developer experience during this release cycle by introducing Tilt to the developer workflow.
+Using Tilt enables developers to make changes to Velero and its plugins, and have those changes automatically built and deployed to your cluster.
+This removes the need for any manual building or pushing of images, and provides a faster and much simpler workflow.
+Our Tilt configuration also enables contributors to more easily debug the Velero process using Delve, which has integrations with many editors and IDEs.
+If you would like to try it out, please see our [documentation](https://velero.io/docs/v1.6/tilt/).o
```
Comment (carlisia) [inline site/content/posts/2021-04-13-Velero-1.6-Bring-All-Your-Credentials.md:89]: This `.o` at the end of the paragraph.

**Review by carlisia — CHANGES_REQUESTED** [review]
Looks great Bridget, thank you for volunteering to write this!

Just one stupid typo and otherwise lgtm!

**Review by a-mccarthy — DISMISSED** [review]
LGTM, aside from the typo Carlisia mentioned

File: site/content/posts/2021-04-13-Velero-1.6-Bring-All-Your-Credentials.md (hunk 1 above)
Comment (zubron) [reply site/content/posts/2021-04-13-Velero-1.6-Bring-All-Your-Credentials.md]: :woman_facepalming: haha, I don't know how I added that! Fixed.

**Review by a-mccarthy — APPROVED** [review]
LGTM

**Review by carlisia — APPROVED** [review]

**Pull request merged**